utils/UniswapPoolsHelpers.ts

```diff
@@ -3,8 +3,23 @@ import type { HardhatRuntimeEnvironment } from "hardhat/types";
 import type { IERC20 } from "../typechain/IERC20";
 import { getBdx, getWeth, getWbtc, getUniswapPairOracle, getBot, getAllBDStables } from "./DeployedContractsHelpers";
 import * as constants from "../utils/Constants";
+import { Recorder } from "./Recorder/Recorder";
+import { toRc } from "./Recorder/RecordableContract";
 import { getListOfSupportedLiquidityPools } from "../utils/Constants";
 
+export async function recordUpdateUniswapPairsOracles(hre: HardhatRuntimeEnvironment, recorder: Recorder) {
+  const pools = await getPools(hre);
+  const promises = pools.map(pool => recordUpdateOracle(hre, pool[0].name, pool[1].name, recorder));
+
+  await Promise.allSettled(promises);
+}
+
+export async function recordUpdateOracle(hre: HardhatRuntimeEnvironment, symbol0: string, symbol1: string, recorder: Recorder) {
+  const oracle = toRc(await getUniswapPairOracle(hre, symbol0, symbol1), recorder);
+  await oracle.record.updateOracle();
+}
+
+// TODO - Cleanup after we release the multisig approach - https://lagoslabs.atlassian.net/browse/LAGO-864
```

Comment:
My bad for adding it, to begin with. We can't delete these functions I added the TODO on.

We'll always need both options - using the recorder or using it without it since there is no timelock contract on the chain.

That could apply to both local development (you don't want to wait 1 day for a change) and also on testnets (Arbitrum for example doesn't have Gnosis safe on it, so no point on having the timelock with a minimum of 1 day there..).

I don't want us to have double tasks from now on - one fo the recorder and one without it.

Maybe we should combine it together and have somewhere a setting telling which chain ids support the recorder and which not?
A place for that could be in the file `Constants.ts` after adding the setting to the `constants.interface.ts` file.

WDYT?

Reply:
On second thought, we can just map which strategies to use in each chain and always use the recorder, right?
So on chains with timelock we'll use this strategy, while on chains without it we'll use the one by one strategy (without a multisig).

WDYT?

Reply:
I will work on in on Monday

Reply:
It's done